File: backend-exam/src/main/java/com/backend_exam/backend_exam/dtos/ExamDTO.java
package com.backend_exam.backend_exam.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ExamDTO {
    private String examName;
    private Long categoryId;
    private Long levelId;
    private String pictures;

}

File: frontend/frontend-exam/src/app/dtos/exam.dto.ts
import { CategoryDto } from './category.dto';
import { LevelDto } from './level.dto';
export interface ExamDto {
  examId: number;
  categoryId: number;
  levelId: number;
  examName: string;
  pictures: string;
  createdAt: string;
  createdUp: string;
  category?: CategoryDto;
  level?: LevelDto;
}
